catroid/src/org/catrobat/catroid/physics/PhysicsLook.java

```diff
@@ -42,15 +45,16 @@ public PhysicsLook(Sprite sprite, PhysicsWorld physicsWorld) {
 		physicsObject = physicsWorld.getPhysicsObject(sprite);
 	}
 
-	@Override
-	public void setVisible(boolean visible) {
-		super.setVisible(visible);
-		updatePhysicsObjectState(true);
-	}
 
 	@Override
 	public void setTransparencyInUserInterfaceDimensionUnit(float percent) {
-		super.setTransparencyInUserInterfaceDimensionUnit(percent);
+		if (percent < 0f) {
+			percent = 0f;
+		} else if (percent >= 100f) {
+			percent = 100f;
+		}
+		alpha = (100f - percent) / 100f;
```

Comment:
Duplicate code :scream_cat: (I know why you copied it but should still remove and fix it)